desktop/faqs.md

```diff
@@ -0,0 +1,215 @@
+---
+description: Frequently asked questions
+keywords: desktop, mac, windows, faqs
+redirect_from:
+- /mackit/faqs/
+- /docker-for-mac/faqs/
+- /docker-for-windows/faqs/
+title: Frequently asked questions
+toc_max: 2
+---
+
+## General
+
+### What are the system requirements for Docker Desktop?
+
+For information about Docker Desktop system requirements, see [Docker Desktop for Mac system requirements](../docker-for-mac/install.md#system-requirements) and [Docker Desktop for Windows system requirements](../docker-for-windows/install.md#system-requirements).
+
+### What is an experimental feature?
+
+{% include experimental.md %}
+
+### Where can I find information about diagnosing and troubleshooting Docker Desktop issues?
+
+You can find information about diagnosing and troubleshooting common issues in the Troubleshooting topic. See [Mac Logs and Troubleshooting](../docker-for-mac/troubleshoot.md) topic and Windows Logs and [Windows Logs and Troubleshooting](../docker-for-windows/troubleshoot.md).
+
+If you do not find a solution in Troubleshooting, browse issues on
+[docker/for-mac](https://github.com/docker/for-mac/issues){: target="_blank" rel="noopener" class="_"} or [docker/for-win](https://github.com/docker/for-win/issues){: target="_blank" rel="noopener" class="_"} GitHub repository, or create a new one.
+
+### How do I connect to the remote Docker Engine API?
+
+To connect to the remote Engine API, you might need to provide the location of the Engine API for Docker clients and development tools.
+
+Mac users can connect to the Docker Engine through a Unix socket: `unix:///var/run/docker.sock`.
+
+If you are working with applications like [Apache Maven](https://maven.apache.org/){: target="_blank" rel="noopener" class="_"}
+that expect settings for `DOCKER_HOST` and `DOCKER_CERT_PATH` environment
+variables, specify these to connect to Docker instances through Unix sockets.
+
+For example:
+
+```bash
+export DOCKER_HOST=unix:///var/run/docker.sock
+```
+
+Docker Desktop Windows users can connect to the Docker Engine through a **named pipe**: `npipe:////./pipe/docker_engine`, or **TCP socket** at this URL:
+`tcp://localhost:2375`.
+
+This sets `DOCKER_HOST` and `DOCKER_CERT_PATH` environment variables to the
+given values (for the named pipe or TCP socket, whichever you use).
+
+See also [Docker Engine API](../engine/api/index.md) and Docker Desktop for Mac forums topic
+[Using pycharm Docker plugin](https://forums.docker.com/t/using-pycharm-docker-plugin-with-docker-beta/8617){: target="_blank" rel="noopener" class="_"}.
```

Comment:
This is a post from 2016, I think we should probably remove this.
They are probably a lot of libraries in many different languages these days.